.github/dependabot.yml

```diff
@@ -0,0 +1,216 @@
+version: 2
+updates:
+  - package-ecosystem: "pub"
+    directory: "/packages/cqrs/"
+    groups:
+      all_dependencies:
+        patterns:
+          - "*"
+    schedule:
+      interval: "daily"
+    versioning-strategy: widen
+    labels:
+      - "Dependencies update"
+      - "p: cqrs"
```

Comment:
I dont think we need any labels. A "dependencies" label is added automatically by dependabot. The package label should be handled by a auto-labeler anyways